### PR TITLE
361/formatting

### PR DESCRIPTION
**Pull request by dpipemazo**
**Infrastructure Changes**

- Creates new `formatter` container which contains `flake8` and `black`
- Adds build + deploy of `formatter` to CircleCI
- Adds formatting check to **all** python code in the Atom repo
- Adds documentation on how to use/check formatting to the README

**Code Changes**
- Formats code with `black`
- Updates code to comply with ER `.flake8`

### CODE REVIEW

**Review by plipay — APPROVED** [review]
Looks great, thanks for taking the initiative on this.
I'd maybe also toss in the docker run cmd example you mentioned on slack earlier into the README: `docker run -e DO_FORMAT=y --mount src=$(pwd),target=/code,type=bind elementaryrobotics/formatter` , so there's an easy copy-paste 1-liner folks can do without having to clone the repo

File: languages/python/atom/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from atom.element import Element, AtomError, MetricsLevel
+from atom.element import Element, AtomError, MetricsLevel  # noqa F401
```
Comment (Cody-G) [inline languages/python/atom/__init__.py:1]: Is this comment something that the formatter added?

**Review by Cody-G — APPROVED** [review]
Looks great, agreed with Peter would be nice to have that docker run command in the README as well.  I just marked one weird comment, wondering if that was intentional or added by Black

File: languages/python/atom/__init__.py (hunk 1 above)
Comment (dpipemazo) [reply languages/python/atom/__init__.py:1]: No, I added this since flake8 complains about it being unused otherwise

Comment (Cody-G) [reply languages/python/atom/__init__.py:1]: Ah cool, didn't know we could exclude with comments like that

Comment (dpipemazo) [reply languages/python/atom/__init__.py:1]: Yeah, also not something to make a habit of using :smile:

**Pull request merged**